src/sentry/static/sentry/app/index.js

```diff
@@ -4,6 +4,7 @@ import 'bootstrap/js/alert';
 import 'bootstrap/js/tab';
 import 'bootstrap/js/dropdown';
 
+import 'app/publicPath';
```

Comment:
can we move this to utils like `emotion-setup` below

Reply:
Yeah sounds good